src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -1146,6 +1154,46 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/request-content-url':
+    post:
+      x-alfresco-since: "7.0"
```

Comment:
Should this be 7.0 or 7.0.0 ?

If 7.0 then we might need to go back and fix others (Version Renditions API) for consistency ?

Reply:
I updated to 7.0.0, thanks for spotting that.